services/QuillLMS/client/app/bundles/PremiumHub/containers/diagnosticGrowthReports/helpers.tsx

```diff
@@ -448,11 +448,11 @@ function renderEmbeddedTable(aggregate_rows) {
       <table className='embedded-student-growth-diagnostic-table'>
         <thead>
           <tr>
-            <th className="diagnostic-skill">Diagnostic Skills</th>
-            <th className="pre-questions-correct">Pre: Questions Correct</th>
-            <th className="pre-skills-status">Pre: Skills Status</th>
-            <th className="post-questions-correct">Post: Questions Correct</th>
-            <th className="post-skills-status">Post: Skill Status</th>
+            <th aria-label="Diagnostic Skills" className="diagnostic-skill">Diagnostic Skills</th>
+            <th aria-label="Pre: Questions Correct" className="pre-questions-correct">Pre: Questions Correct</th>
+            <th aria-label="Pre: Skills Status" className="pre-skills-status">Pre: Skills Status</th>
+            <th aria-label="Post: Questions Correct" className="post-questions-correct">Post: Questions Correct</th>
+            <th aria-label="Post: Skill Status" className="post-skills-status">Post: Skill Status</th>
```

Comment:
No need to remove these, but just for future reference, it's okay to not have aria-labels for `<th />`s if the content is reasonably succinct. It's more like when we have a bunch of tooltips and stuff within them and it reads all the text of all of those that it becomes too much for the screenreader IMO.

Reply:
ok! good to know, thanks for explaining